yarn.lock

```diff
@@ -0,0 +1,1107 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
At Yarn, they recommend to check it:
https://yarnpkg.com/en/docs/version-control#toc-required-files

Is there any reason not to follow their recommendation?